tests/agent/conftest.py

```diff
@@ -24,6 +24,8 @@ def test_id():
 def local_config(test_id, etcd_container, redis_container):  # noqa: F811
     ipc_base_path = Path.cwd() / f'.tmp/{test_id}/agent-ipc'
     ipc_base_path.mkdir(parents=True, exist_ok=True)
+    var_base_path = Path.cwd() / 'var/lib/backend.ai'
```

Comment:
For testing, we need to use an isolated configuration other than the current development sources, like the above `ipc_base_path`'s using the `test_id` fixture.

```suggestion
    var_base_path = Path.cwd() / '.tmp/{test_id}/agent-var'
```